src/tracing/BUILD.gn

```diff
@@ -35,6 +35,8 @@ config("multiplexed_tracing") {
 static_library("tracing") {
   sources = [
     "backend.h",
+    "group_keys.h",
+    "label_keys.h",
```

Comment:
if I do not tracke groups or labels, I should not need these files. These should be in separate build targets. It also seems that these are tied to session/group and such and not directly related to tracing.

otherwise we are coupling a generic tracing functionality with what tracing is possible (right now groups and labels)

Reply:
This could be changed in the tracing backend, the current state of the sdk is that all the macros calls have a (label, group) arguments, in what use case could we enable matter tracing without supporting groups and labels?